### DISCUSSION TITLE
Line spacing in EPUB or FB2

### DISCUSSION THREAD

**Discussion by VeslavG** [sumatrapdfreader/sumatrapdf#3201] [Q&A]
Hello, 

Thanks for the excellent tool for PDF reading.
I have found that Sumatra works fantastic for EPUB as well. Even better sometimes than Calibre embedded reader.
The only issue I can't solve - is how to set line spacing. I understand that it is not possible for PDFs (reflow is needed).
But it surely should be possible for EPUBs.
Could you please advise how to manage line spacing for EPUBs in Sumatra?

Many thanks!

**Comment by GitHubRulesOK** [comment]
line spacing is set by the author and an editor like Calibre can edit the value whereas SumatraPDF now relies on MuPDF which as reader does not edit the content, although in MuPDF there is a scale lines up down by key press, that is one of the features not used in SumatraPDF (it's on its own not much use as it simply reflows as bigger or smaller). There is also in MuPDF an ability to over-ride some css but again that's very document specific display orientated.

**Comment by VeslavG** [comment]
Thanks, I've thought it is easier to make/implement. This is because basically each ebook (Kindle, Onyx, Inkpad, ...) provides very granular possibilities to control a "reflow" of EPUB (MOBI, AZW3, ...)  - including line spacing.

I've thought it is just a matter of finding this setting in Sumatra somewhere. :(

**Reply by GitHubRulesOK** [reply]
It was possible in legacy versions to use a different font thus inter character and interline spacing would be affected as a secondary consequence, however current MuPDF engin  "tries" (sometimes fails) to match the authored intents ! but currently no adjustments have been added to that other than perhaps one fixed page size.

**Reply by daoudiAli** [reply]
But why its possible in android version to space the lines as we want in many  Epub reading apps

**Reply by daoudiAli** [reply]
https://www.mobileread.com/forums/showthread.php?t=27562
